### ISSUE TITLE
Wifi State Change Callback

### ISSUE THREAD

**Issue by sunnykinger** [ThanosFisherman/WifiUtils#1]
Can't we have some callback method that can give us wifi state change status?

**Comment by ThanosFisherman** [comment]
Yes I was thinking about it too. Since this library already monitors wifi state, I will probably expose those constants in callback at the next release.

**Comment by fukemy** [comment]
Hello.  Thanks you for great library.
 I am developing application that connect to hotspot then using socket to communicate with hardware, that's post is important to detect when network is break or something similar...
 Now is 2018, does this project support this?

**Comment by ThanosFisherman** [comment]
Hello @fukemy 
This library helps you indeed connect to hotspots but it doesn't monitor connectivity changes. You will need to implement this yourself.
